src/sssom/writers.py

```diff
@@ -547,7 +617,7 @@ def get_writer_function(
         return write_rdf, SSSOM_DEFAULT_RDF_SERIALISATION
     elif output_format == "json":
         return write_json, output_format
-    elif output_format == "fhir_json":
+    elif output_format == "fhir":
```

Comment:
@matentzn Bugfix. Right now we're on `sssom==0.3.40`. I had a copy of `0.3.19` where this wasn't the case, so it happened some time after that.

I needed to make this change to match with `SSSOM_EXPORT_FORMATS = ["tsv", "rdf", "owl", "json", "fhir", "ontoportal_json"]`, otherwise the CLI was not working for `fhir`.

<details><summary>Additional details</summary>
<p>

The problem was like this. I tried `fhir_json` but it was angry:
`sssom convert content/minimal_example/tsv/v2/example.sssom.tsv --output-format fhir_json --output content/minimal_example/fhir/example.fhir.json`
```
Usage: sssom convert [OPTIONS] INPUT
Try 'sssom convert --help' for help.
Error: Invalid value for '-O' / '--output-format': 'fhir_json' is not one of 'tsv', 'rdf', 'owl', 'json', 'fhir', 'ontoportal_json'.
```

It says above that `fhir` was OK, but when I used it, it didn't work.
`sssom convert content/minimal_example/tsv/v2/example.sssom.tsv --output-format fhir --output content/minimal_example/fhir/example.fhir.json`
`ValueError: Unknown output format: fhir`

</p>
</details>

Reply:
FYI, there's another, unrelated CLI issue: 
- #410